src/modules/musig/session_impl.h

```diff
@@ -450,7 +447,7 @@ int secp256k1_musig_nonce_process(const secp256k1_context* ctx, secp256k1_musig_
         }
         secp256k1_gej_add_ge_var(&aggnonce_ptj[0], &aggnonce_ptj[0], &adaptorp, NULL);
```

Comment:
Thanks for the PR! I think you need to take this line into account. The [0] aggnonce may have changed, so we need to convert the changed nonce back to ge. (I don't think the same is necessary for [1])

Reply:
Moved the adaptor logic to the internal function.

1) When the adaptor is NULL, there is no need to convert between `gej` and `ge`.
2) When the adaptor is not NULL, we only convert the required aggnonce[0]